### PR TITLE
fix: Corrected the upload_workspace api

### PR DESCRIPTION
**Pull request by abhilash-kumar-nair**
This PR fixes the upload workspace API.

### CODE REVIEW

**Review by jacobtaxen — APPROVED** [review]
Looks good and works fine

File: modelon/impact/client/sal/service.py [hunk 1]
```diff
@@ -558,6 +567,26 @@ def data(self):
         return self._resp_obj.content
 
 
+class FormDataResponse(Response):
```
Comment (efredriksson-modelon) [inline modelon/impact/client/sal/service.py:570]: This class is not used anywhere?

File: modelon/impact/client/sal/service.py [hunk 2]
```diff
@@ -341,6 +341,10 @@ def post_json(self, url, body=None, files=None):
         request = RequestJSON(self._context, "POST", url, body, files)
         return request.execute().data
 
+    def post_form_data(self, url):
```
Comment (efredriksson-modelon) [inline modelon/impact/client/sal/service.py:344]: This method is not used anywhere?

**Review by efredriksson-modelon — CHANGES_REQUESTED** [review]
I am a little confused as the 'Form' classes/methods are not used.

**Comment by abhilash-kumar-nair** [comment]
> I am a little confused as the 'Form' classes/methods are not used.

@efredriksson-modelon , You are right! The response body would be still be a JSON, so the only issue was  the additional workspace Id argument . I have made the correction now. Thanks!

**Review by efredriksson-modelon — APPROVED** [review]
👍

**Pull request merged**